axum-extra/CHANGELOG.md

```diff
@@ -7,9 +7,15 @@ and this project adheres to [Semantic Versioning].
 
 # Unreleased
 
+- None.
+
+# 0.3.3 (18. May, 2022)
+
 - **added:** Add `extract::Query` which supports multi-value items ([#1041])
+- **added:** Support customizing rejections for `#[derive(TypedPath)]` ([#1012])
```

Comment:
I suppose we need to ship an axum-macros update first and then update axum-extra to depend on that version.

Reply:
Oh, right!